build/common.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <Import Project="..\version.props" />
+  <Import Project="..\.build\common.props" Condition="Exists('..\.build\common.props')" />
+
+  <PropertyGroup>
+    <RepositoryUrl>https://github.com/aspnet/PlatformAbstractions</RepositoryUrl>
```

Comment:
yeah, I caught that later :) 

https://github.com/aspnet/Common/commit/27b47956ebf045ec84a8b51634ed9706bd6ea3a5#diff-9780bf7a72094f1ac4b023e438b548ba